docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
    redis:
-      image: 'redis:3.2-alpine'
+      image: 'redis:5.0.3-alpine'
```

Comment:
please, also add:

```yml
ports:
  - "6379:6379"
```

to enable connecting to redis when running the server from host

Reply:
I think this not a good practice since you will expose redis on port 6379 and anyone can mess up your redis server unless you add a very strong password but still not recommended

the best options is to use
```yml
version: '3'
services:
   redis:
      image: 'redis:3.2-alpine'
      volumes:
         - 'redis_data:/data'
      restart: always
   web:
      image: "clivern_beaver:1.1.1"
      build: .
      command: './beaver'
      ports:
         - "8080:8080"
      depends_on:
         - redis
      volumes:
         - './var:/go/src/github.com/clivern/beaver/var'
      restart: always
      healthcheck:
         test: './beaver -exec=health'
         interval: 5s
         timeout: 2s
         retries: 5
volumes:
    redis_data: null
```
since you will be able to access redis server from the web service with url as redis:6379 or use networks

```yml
version: '3'
services:
   redis:
      image: 'redis:3.2-alpine'
      volumes:
         - 'redis_data:/data'
      restart: always
    networks:
      - redis-net
   web:
      image: "clivern_beaver:1.1.1"
      build: .
      command: './beaver'
      ports:
         - "8080:8080"
      depends_on:
         - redis
      volumes:
         - './var:/go/src/github.com/clivern/beaver/var'
      restart: always
      healthcheck:
         test: './beaver -exec=health'
         interval: 5s
         timeout: 2s
         retries: 5
    networks:
      - redis-net
volumes:
    redis_data: null
networks:
  redis-net:
```

Reply:
I see your point. Thanks.

I thought the idea here was to use docker-compose.yml only for development purpose. Anyway if you run `docker-compose up redis` and then run the tests or the server itself from the host you won't be able to connect to redis since the port 6379 is not open for connections from outside docker.

Reply:
yeah this can be a usage case, anyway i updated the file and commented it
```
      # Allow connection from docker host machine
      # ports:
      #   - "6379:6379"
```